os/include/debug.h

```diff
@@ -845,6 +845,22 @@
 #define sdllvdbg(...)
 #endif
 
+#ifdef CONFIG_DEBUG_SECURITY_ISP_WRAPPER_ERROR
+#define sidbg(format, ...)     dbg(format, ##__VA_ARGS__)
```

Comment:
ISP may be vendor-specific solution but sidbg and its variants should be defined in common for all vendors.

Reply:
That is very critical comment. You are right. However, it is hard to keep printf in this kernel code. Is it okay to put dbg instead of newly defined sidbg here?  Please let me know your opinion.

Reply:
We newly define DEBUG_SECURE_ELEMENT_* and applied them. Please let me know your opinion. Thanks!